server/src/main/java/org/opensearch/gateway/GatewayMetaState.java

```diff
@@ -611,6 +612,11 @@ public void setLastAcceptedState(ClusterState clusterState) {
             lastAcceptedState = clusterState;
         }
 
+        @Override
+        public PersistedStateStats getPersistedStateStats() {
+            return null;
```

Comment:
Add a note that for LucenePersistence, these stats are not published yet.

Reply:
Added.